app/org/maproulette/data/DataManager.scala

```diff
@@ -411,10 +410,39 @@ class DataManager @Inject()(config: Config, db: Database, boundingBoxFinder: Bou
                 filter.append(" OR t.id NOT IN (SELECT task_id FROM task_review tr WHERE tr.task_id = t.id)")
               }
               filter.append(")")
-              reviewStatusFilter = filter.toString()
+              searchFilters.append(filter)
             case Some(statuses) if statuses.isEmpty => //ignore this scenario
             case _ =>
           }
+
+          search.owner match {
+           case Some(o) if o.nonEmpty =>
+             val filter = new StringBuilder(s""" AND (t.id IN (SELECT task_id
```

Comment:
Is there any particular you are creating a StringBuilder object here instead of just putting the string as the parameter in the append function?

Reply:
No good reason, I was just following the prior pattern. I've removed it.